bundles/org.openhab.binding.solarwatt/src/main/java/org/openhab/binding/solarwatt/internal/SolarwattConfiguration.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.solarwatt.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link SolarwattConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Sven Carstens - Initial contribution
+ */
+@NonNullByDefault
+public class SolarwattConfiguration {
+    private static final int DEFAULT_RESCAN_MINUTES = 5;
+    private static final int DEFAULT_REFRESH_SECONDS = 30;
+
+    /**
+     * Hostname or ip where the solarwatt energymanager is reachable.
+     */
+    public String hostname = "default name";
```

Comment:
```suggestion
    public String hostname = "";
```
I did a search and could not find that you used that value anywhere, can be be made a empty string to keep the compiler happy?

Reply:
As there seems to be now reasonable default value, I changed it to be the empty string.